code/modules/admin/verbs/randomverbs.dm

```diff
@@ -853,3 +853,47 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		to_chat(usr, "Random events disabled")
 		message_admins("Admin [key_name_admin(usr)] has disabled random events.", 1)
 	feedback_add_details("admin_verb","TRE") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	
+	
+/client/proc/cmd_admin_cryo(mob/living/M as mob in SSmobs.mob_list)
+	set category = "Special Verbs"	
+	set name = "Admin Cryo"
+	if(!check_rights(R_MOD))
```

Comment:
using usr in a proc is bad, I suggest using !check_rights(R_MOD, 1, src), which also automatically gives failure message